custom/icds_reports/management/commands/sql_scripts/fix_past_data_part_3.sql

```diff
@@ -0,0 +1,23 @@
+UPDATE "agg_child_health_%(start_date)s_5" agg_child_health
+  SET
+    wasting_moderate = ut.wasting_moderate,
+    wasting_severe = ut.wasting_severe,
+    wasting_normal = ut.wasting_normal,
+    stunting_moderate = ut.stunting_moderate,
+    stunting_severe = ut.stunting_severe,
+    stunting_normal = ut.stunting_normal,
+    height_measured_in_month = ut.height_measured_in_month,
+    weighed_and_height_measured_in_month = ut.weighed_and_height_measured_in_month
+  FROM (
```

Comment:
this would be much clear as `FROM temp_agg_child_my` without the unnecessary subselect

Reply:
Done